src/main/java/com/iexec/sms/admin/AdminService.java

```diff
@@ -149,4 +143,72 @@ boolean restoreDatabaseFromBackupFile(String storageLocation, String backupFileN
         }
         return false;
     }
+
+    /**
+     * Delete a backup of the H2 database from a location
+     *
+     * @param storageLocation The location of the backup file.
+     * @param backupFileName  The name of the backup file.
+     * @return {@code true} if the deletion was successful; {@code false} if any error occurs.
+     */
+    public boolean deleteBackupFileFromStorage(String storageLocation, String backupFileName) {
+        try {
+            // Ensure that storageLocation and backupFileName are not blanks
+            boolean validation = commonsParametersValidation(storageLocation, backupFileName);
+            if (!validation) {
+                return false;
+            }
+            // Ensure that storageLocation ends with a slash
+            storageLocation = normalizePathWithSeparator(storageLocation);
+            String fullBackupFileName = storageLocation + backupFileName;
+
+            final File backupFile = new File(fullBackupFileName);
+            final String backupFilePath = backupFile.getCanonicalPath();
+            // Ensure that storageLocation correspond to an authorised area
+            if (!backupFilePath.startsWith(adminStorageLocation)) {
+                throw new IOException("Backup file is outside of storage file system");
+            } else if (!backupFile.exists()) {
+                throw new IOException("Backup file does not exist");
```

Comment:
For consistency with restore endpoint (even if it may change due to a rework, I think it is a good practice to stay as consistent as possible) ?
```suggestion
                throw new FileSystemNotFoundException("Backup file does not exist");
```

Reply:
locally modified, thanks!